secure_message/services/party_service.py

```diff
@@ -10,18 +10,31 @@
 
 
 class PartyService:
+
+    __users_cache = dict()
+    __business_details_cache = dict()
+
     @staticmethod
-    def get_business_details(ru):
-        """Retrieves the business details from the party service"""
+    def get_url(api_param, code):
+        """
+        :param api_param:  is the key configuration that represent part or the URI .
```

Comment:
To stay consistent with the rest of the docstrings, it's probably worth dropping the space and "." from the end of this line.